components/AgentEconomies/MechPage/MechAgentMetrics.jsx

```diff
@@ -108,54 +113,84 @@ export const MechAgentMetrics = () => {
   );
 
   return (
-    <SectionWrapper
-      customClasses="text-center py-16 px-4 border-b border-t"
-      id="stats"
-    >
-      <div className="text-7xl lg:text-9xl mb-12 max-w-[650px] mx-auto mb-16">
-        <p className="text-xl text-slate-700 mb-8 mx-auto">
-          The Olas Mech agent economy is thriving, powering over{' '}
+    <SectionWrapper customClasses="text-center py-16 border-t" id="stats">
+      <div className="text-7xl lg:text-9xl mb-12 max-w-[650px] mx-auto mb-16 w-full">
+        <Card className="flex flex-col gap-6 p-8 mx-auto border border-purple-200 rounded-full text-xl w-fit rounded-2xl bg-gradient-to-t from-[#F1DBFF] to-[#FDFAFF] items-center">
+          <div className="flex items-center">
+            <Image
+              alt="Mech DAAs"
+              src="/images/agents/mech.svg"
+              width="35"
+              height="35"
+              className="mr-4"
+            />
+            Mech Agent Economy
+          </div>
+          {metrics?.DAAs ? (
+            <Link className="font-extrabold text-6xl" href="/data" hideArrow>
```

Comment:
should also have explanation for this query on data page and link to relevant section